config/initializers/appsignal.rb

```diff
@@ -0,0 +1,30 @@
+require 'open_project/version'
+
+if ENV['APPSIGNAL_ENABLED'] == 'true'
+  require 'appsignal'
+  OpenProject::Application.configure do |app|
+    Appsignal.config = Appsignal::Config.new(
+      Rails.root,
+      Rails.env, {
+        active: true,
+        name: ENV.fetch('APPSIGNAL_NAME'),
+        push_api_key: ENV.fetch('APPSIGNAL_KEY'),
+        revision: OpenProject::VERSION.to_s,
+        log: 'stdout',
+        log_level: 'debug',
```

Comment:
We should either use the application's log level or make this configurable through the environment separately and set to  warning/error by default.